templates/checkout/_partials/cart-voucher.tpl

```diff
@@ -34,7 +34,7 @@
               <li class="cart-voucher__item row">
                 <span class="cart-voucher__name col">{$voucher.name}</span>
                 <div class="d-flex align-items-center justify-content-end col">
-                  <span class="me-2">{$voucher.reduction_formatted}</span>
+                  <span class="me-2 fw-bold">{$voucher.reduction_formatted}</span>
```

Comment:
@NeOMakinG Maybe ms-2 on the other element? If it's not deletable, there will be space.